src/components/ContentNode/CodeListing.vue

```diff
@@ -166,6 +166,7 @@ pre {
   white-space: pre;
   word-wrap: normal;
   height: 100%;
+  tab-size: var(--code-indentationWidth, 4);
```

Comment:
Would it be better for this to be `--code-indentation-width`?

Reply:
Probably, from a style perspective.

That's just the way that the existing code works for transforming theme settings keys into CSS property names where it just uses the JSON key as-is and doesn't adjust it for use in CSS in terms of a different naming style. I didn't want to mess with that code or the existing API.

Reply:
https://github.com/apple/swift-docc-render/blob/2d0c69878ff48d798aaac04db3c7083132edc4bb/src/utils/themes.js#L49